examples/offboard_velocity/offboard_velocity.cpp

```diff
@@ -259,6 +259,17 @@ int main(int argc, char** argv)
     std::cout << "In Air..." << std::endl;
     sleep_for(seconds(5));
 
+    while (true) {
```

Comment:
I would find it more elegant to use [landed_state_async](https://github.com/mavlink/MAVSDK/blob/develop/src/plugins/telemetry/include/plugins/telemetry/telemetry.h#L1196) with a future/promise to wait until the right state is reached.

That would get rid of the `while (true)` and active waiting (`sleep_for(milliseconds(500))`)...

Reply:
Agreed,

Reply:
And since you added the wait, you can remove the `sleep_for(seconds(5))` above.